src/org/etools/j1939_84/controllers/part04/Part04Step15Controller.java

```diff
@@ -53,12 +57,51 @@
     @Override
     protected void run() throws Throwable {
         // 6.4.15.1.a Turn the engine off.
+        incrementProgress("Part 4, Step 15 Turn Engine Off and keep the ignition key in the off position");
```

Comment:
On the progress, let's use "Step 4.15.  ....."  That's consistent with the start/end part messages.  Additionally, I'm going to add those numbers to _every_ progress update so the progress bar does inform the user on what's happening

Reply:
Okay, but the current pattern on these is as shown.  This was a direct copy of Part 03 Step 16.  Do we want to enter a ticket for this work?

Reply:
We don't need to enter a ticket to go back and fix what's already been done.  But if you find it and want to fix it that's good.  Going forward we should use Step X.Y. notation to give the UI a consistent L&F.